torchtext/datasets/sst.py

```diff
@@ -0,0 +1,45 @@
+from torchtext.utils import download_from_url, extract_archive
+from torchtext.datasets.common import RawTextIterableDataset
+from torchtext.datasets.common import wrap_split_argument
+from torchtext.datasets.common import add_docstring_header
+from torchtext.datasets.common import find_match
+import os
+import io
+try:
+    from nltk.tree import Tree
+except ImportError:
+    print("Please install NLTK. See the docs at https://nltk.org for more information.")
```

Comment:
I'd raise this as an error, so that processing doesn't continue. And further I'd make explicit that this is due to this dataset specifically.